src/main/java/com/jenkinsci/plugins/badge/BadgePlugin.java

```diff
@@ -1,67 +0,0 @@
-/*
- * The MIT License
- *
- * Copyright (c) 2024, Badge Plugin Authors
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-package com.jenkinsci.plugins.badge;
-
-import edu.umd.cs.findbugs.annotations.NonNull;
-import hudson.Extension;
-import hudson.model.PersistentDescriptor;
-import jenkins.model.GlobalConfiguration;
-import jenkins.model.GlobalConfigurationCategory;
-import org.kohsuke.stapler.DataBoundSetter;
-
-@Extension
-public class BadgePlugin extends GlobalConfiguration implements PersistentDescriptor {
-
-    private boolean disableFormatHTML;
```

Comment:
See original PR:

> HTML formatting delegated to global Jenkins MarkupFormatter

> Remove Badge Plugin global configuration
> All text is formatted using Jenkins.get().getMarkupFormatter().translate()
> Breaking: It is no longer possible to use non-sanitized HTML in badges (unless there is a MarkupFormatter for it, that I'm unaware of)

Since the configuration is removed it will likely break JCaC - not sure how non-existant configurations are being handled to be honest. I would not know any way around it so it will be documented as breaking change as it has already been done in the past.

Reply:
But, I does not see braking change in PR description